src/driver/analysis_example_experimental_design.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Experimental design for a quick eval to establish analysis practice."""
+
+from absl import app
+from absl import flags
+import itertools
+import math
+import numpy as np
+from pyDOE import lhs
```

Comment:
Are `lhs`, `app` and `flags` used anywhere?

Reply:
Ditto.